SETUP.md

```diff
@@ -20,6 +20,48 @@ Instructions are available in our [wiki](https://github.com/CircuitVerse/Circuit
 
 **Note**: Please follow [these](https://docs.docker.com/docker-for-windows/install-windows-home/) instructions for installing Docker on Windows 10 Home
 
+Installation for ubuntu linux(also for ubuntu based)
+
+Install docker and docker-compose
+
+sudo apt install docker docker-compose
+
+Enable docker service
+
+sudo systemctl enable docker.service
+sudo systemctl enable containerd.service
+
+Start docker service
+
+sudo systemctl start docker.service
+sudo systemctl start containerd.service
+
+Add docker usergroup
+
+Creating docker user-group
+
+sudo groupadd docker
+
+Add docker group to your User group
+
+sudo usermod -aG docker $USER
+
+Logout and Login
+
+Check the installation
+
+Run
+
+docker info
+
+docker run hello-world
+
+Check versions
+
+docker --version
+docker-compose --version
+
+
```

Comment:
```suggestion
### Setup for MacOS
```

Reply:
I don't have a mac.Please guide me how?

Reply:
refer here https://docs.docker.com/docker-for-mac/apple-m1/

Reply:
Thanks. I'll check it out .

Reply:
Hey sorry I am not familiar with brew infact never used it and creating docs similar to one present on official website won't
be a good idea. Since I was familiar with apt so did the ubuntu part.